src/cheri/riscv-hybrid-integration.adoc

```diff
@@ -255,30 +262,30 @@ IMPORTANT: need to resolve how to cross-reference the privileged spec to have a
 //determined (see <<extended_CSR_writing>>).
 //* The full capability is written to destination *c* register.
 
-When any CLEN-width CSR is accessed with another CSR instruction:
-
-* <<CSRRWI_CHERI>>
-* <<CSRRC_CHERI>>
-* <<CSRRCI_CHERI>>
-* <<CSRRS_CHERI>>
-* <<CSRRSI_CHERI>>
-
-The following rules apply:
-
-* The final address is calculated according to the standard RISC-V CSR rules (set bits, clear bits etc).
-* The final address is updated as specified in <<extended_CSR_writing,the privileged specification>> for an XLEN write.
-    ** Unless specified otherwise, the update uses <<SCADDR>> semantics
-* When accessing an extended CSR:
-    ** In {cheri_int_mode_name}, XLEN-bits are read from the capability address field and written to an output *x* register.
-    ** In {cheri_cap_mode_name}, CLEN-bits are read from the CSR and written to an output *c* register.
-* When accessing a CSR added by a CHERI extension:
-** CLEN-bits are read from the CSR and written to an output *c* register.
-
-//NOTE: For all non-CLEN-wide CSRs (i.e., XLEN-width CSRs) XLEN-bits are read/written regardless of <<cheri_execution_mode>>.
-
-ifdef::cheri_v9_annotations[]
-NOTE: *CHERI v9 Note:* CSpecialRW is removed. Its role is assumed by <<CSRRW_CHERI>>.
-endif::[]
-
+//When any CLEN-width CSR is accessed with another CSR instruction:
+//
+//* <<CSRRWI_CHERI>>
+//* <<CSRRC_CHERI>>
+//* <<CSRRCI_CHERI>>
+//* <<CSRRS_CHERI>>
+//* <<CSRRSI_CHERI>>
+//
+//The following rules apply:
+//
+//* The final address is calculated according to the standard RISC-V CSR rules (set bits, clear bits etc).
+//* The final address is updated as specified in <<extended_CSR_writing,the privileged specification>> for an XLEN write.
+//    ** Unless specified otherwise, the update uses <<SCADDR>> semantics
+//* When accessing an extended CSR:
+//    ** In {cheri_int_mode_name}, XLEN bits are read from the capability address field and written to an output *x* register.
+//    ** In {cheri_cap_mode_name}, CLEN bits are read from the CSR and written to an output *c* register.
+//* When accessing a CSR added by a CHERI extension:
+//** CLEN bits are read from the CSR and written to an output *c* register.
+//
+//NOTE: For all non-capability CSRs (i.e., XLEN-width CSRs) XLEN bits are read/written regardless of <<cheri_execution_mode>>.
+//
+//ifdef::cheri_v9_annotations[]
+//NOTE: *CHERI v9 Note:* CSpecialRW is removed. Its role is assumed by <<CSRRW_CHERI>>.
+//endif::[]
+//
```

Comment:
I think it would make sense to remove lines we do not need anymore instead of commenting them out.

Reply:
done